src/storage/src/render/debezium.rs

```diff
@@ -85,16 +82,12 @@ where
 
                         if should_use {
                             match value.iter().nth(before_idx).unwrap() {
-                                Datum::List(l) => {
-                                    session.give((Ok(Row::pack(&l)), cap.time().clone(), -1))
-                                }
+                                Datum::List(l) => session.give((Ok(Row::pack(&l)), time, -diff)),
```

Comment:
i still think `-1` and `1` here are more clear than using the `diff` value thats always `1`! just a nit, not required